src-built-in/components/workspaceManagementMenu/src/stores/workspaceManagementMenuStore.js

```diff
@@ -565,7 +567,7 @@ Actions = {
 			if (response.choice === "affirmative") {
 				callback(null, workspaceName);
 			} else {
-				callback("Negative");
+				callback(NEGATIVE);
```

Comment:
we should pick one of return (new Error(NEGATIVE)) or return(NEGATIVE). This is inconsistent

Reply:
Good catch! Done.